### ISSUE TITLE
Fix use of headings through out

### ISSUE THREAD

**Issue by clhenrick** [clhenrick/am-i-rent-stabilized#85]
Currently there are a lot of incorrect uses of heading elements (`<h1>`, `<h2>`, `<h3>`) in the website's pages. The use of heading elements should be corrected so that [they follow sequentially and are not used for sizing text](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes). 

For the main page, consider making each slide a `<section>` element that has a top level `<h2>` heading to make the page more easily scan-able and the content more accessible by users of assistive technology.

**Comment by clhenrick** [comment]
Fixed in #124
